release_creation/bundle/bundle_os.py

```diff
@@ -0,0 +1,16 @@
+import strenum as strenum
+
+
+class BundleOS(strenum):
+    LINUX = "linux"
+    MAC = "mac"
+    WINDOWS = "windows"
+
+
+# these values have been derived by trial and error, as to date
+# there isn't an authoritative source, see:
+# https://packaging.python.org/en/latest/specifications/platform-compatibility-tags/
+PIP_PLATFORM_OS_VALUES = {
```

Comment:
Is `Bundle.OS.WINDOWS` intentionally missing?

Reply:
yes, haven't invested time in getting windows working

Reply:
That must be a bumper sticker somewhere.